db/seeds/seed.js

```diff
@@ -1,34 +1,34 @@
 const db = require("../connection");
 const format = require("pg-format");
 const {
-	convertTimestampToDate,
-	createRef,
-	formatComments,
+  convertTimestampToDate,
+  createRef,
+  formatComments,
 } = require("./utils");
 
-const seed = async (data) => {
-	const { categoryData, commentData, reviewData, userData } = data;
-	await db.query(`DROP TABLE IF EXISTS comments;`);
-	await db.query(`DROP TABLE IF EXISTS reviews;`);
-	await db.query(`DROP TABLE IF EXISTS users;`);
-	await db.query(`DROP TABLE IF EXISTS categories;`);
+const seed = async (categoryData, commentData, reviewData, userData) => {
+  //const { categoryData, commentData, reviewData, userData } = data;
```

Comment:
Originally this seed function was taking 1 argument which had  4 keys  (categoryData, commentData etc), and now you've changed it to take 4 (which is why it's currently now working, when you are calling it with 4 arguments in your test file).

Originally it was making use of destructuring. see:

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment